index.js

```diff
@@ -41,6 +41,9 @@ export default class Breakdancer {
    * @private
    */
   normalize (specification) {
+    if (!specification) {
```

Comment:
I don't really see the need of this change. If there is no specification of breakpoints there would be absolutely no need to use this module in the first place.